lib/timetable-visualisation/PlanViewer.tsx

```diff
@@ -46,33 +47,38 @@ export const PlanViewer = ({
     throw new Error("published event not found");
   }
 
-  const stagesInfo = useMemo<StagePreviewInfo[]>(
-    () => [
-      {
-        name: "Local Development Scheme Published",
-        startEvent: publishedEvent,
-      },
-      ...stages.map<StagePreviewInfo>((stage) => {
+  const stagesInfo = useMemo<StagePreviewInfo[]>(() => {
+    const foundStages = stages.reduce(
+      (allStages: StagePreviewInfo[], currentStage) => {
         const startEvent = timetableEvents.find(
-          (event) => event.developmentPlanEvent === stage.startEventKey
+          (event) => event.developmentPlanEvent === currentStage.startEventKey
         );
 
         const endEvent = timetableEvents.find(
-          (event) => event.developmentPlanEvent === stage.endEventKey
+          (event) => event.developmentPlanEvent === currentStage.endEventKey
         );
 
-        if (!startEvent) {
-          throw new Error("event not found");
+        if (startEvent && startEvent.eventDate) {
```

Comment:
Note: for a stage to be shown we need to have an event matching the stage startEvent and the event has to have an eventDate. This means that we are going to display events with startDate and no endDate but not events with endDate and no startDate

Reply:
Cool, seems somewhat unlikely that someone would put an end date without a start (and we can always review later)